client/src/components/Sub_RatingsReviews/generateStars.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import uuid4 from 'uuid4';
+import Star from './Star.jsx';
+
+export default function generateStars(rating, amtStars, size = 25) {
+  const result = [];
+  let remaining = rating;
```

Comment:
not sure if you need this

Reply:
The linter doesn't like it when you perform actions on the parameter.  😂
That's why I had to be explicit and include line 7.

Reply:
omg linter lol okay!